tests/python/unittest/test_sparse_operator.py

```diff
@@ -53,20 +56,21 @@ def test_elemwise_add_ex_multiple_stages():
     sp_data1 = mx.symbol.Variable('sp_data1', storage_type='row_sparse')
     sp_data2 = mx.symbol.Variable('sp_data2', storage_type='row_sparse')
     ds_data = mx.symbol.Variable('ds_data')
-    plus  = mx.symbol.elemwise_add(sp_data1, sp_data2, name='plus')
+    plus = mx.symbol.elemwise_add(sp_data1, sp_data2, name='plus')
     # sparse + dense = dense
-    test  = mx.symbol.elemwise_add(plus, ds_data)
-    check_symbolic_forward(test, {'sp_data1':sp_nd1, 'sp_data2':sp_nd2,
-                          'ds_data':ds_nd}, [sp_np1 + sp_np2 + ds_np])
+    test = mx.symbol.elemwise_add(plus, ds_data)
+    check_symbolic_forward(test, {'sp_data1': sp_nd1, 'sp_data2': sp_nd2,
+                                  'ds_data': ds_nd}, [sp_np1 + sp_np2 + ds_np])
 
-    arr_grads = [mx.nd.zeros(shape) for i in range(3)]
-    exec_test = test.bind(default_context(), args={'sp_data1':sp_nd1, 'sp_data2':sp_nd2,
-                          'ds_data':ds_nd}, args_grad=arr_grads)
+    arr_grads = [mx.nd.zeros(shape) for i in xrange(3)]
```

Comment:
Sorry, I think it was accidentally changed by PyCharm. Will be careful next time.